apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -152,28 +147,30 @@ class Demosaic : public Halide::Generator<Demosaic> {
         } else if (get_target().has_feature(Target::HVX_128)) {
             vec = 64;
         }
-        for (Func f : intermediates) {
+        auto apply_schedule_to_intermediate = [&](Func& f){
```

Comment:
Since the method this is in is called "schedule" I think we can name lambdas like this "schedule_intermediate" or perhaps "fold_vectorize_and_place" if a more descriptive name seems useful.